### PR TITLE
Fixes for integration bugs in theme and README

### PR DESCRIPTION
**Pull request by gersonnoboa**
Each commit has its description. Please do take a closer look at the delivered status and let me know if my approach to the delivery text is correct. I wasn't able to discern an easier way to do this, as the method that prints this "Delivered" text is used in several different places. I am using a similar approach to what is being used for `AlertConfiguration`.

### CODE REVIEW

**Review by EgorovEI — APPROVED** [review]

**Review by rasmustautsglia — APPROVED** [review]

**Comment by gersonnoboa** [comment]
@igorkravchenko added an unbelievably difficult to write unit test in view model 1d1d3db0b53bf5de14e112a428d4cfea58089710

**Review by igorkravchenko — APPROVED** [review]
Looks good to me 👍

**Comment by gersonnoboa** [comment]
!squash

**Comment by gersonnoboa** [comment]
!merge

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ To integrate `GliaWidgets` into your project via [CocoaPods](https://cocoapods.o
 source 'https://github.com/CocoaPods/Specs.git'
 source 'https://github.com/salemove/glia-ios-podspecs.git'
```
Comment (dukhovnyi) [inline README.md:36]: we could drop this line also since our SDK is available in the official Cocoapods repo...

Comment (gersonnoboa) [reply README.md]: Fixed https://github.com/salemove/ios-sdk-widgets/pull/436/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L36

**Comment by gersonnoboa** [comment]
!merge

**Pull request merged**